### PR TITLE
Blazor Example updated for both Mock and EF core implementation

### PR DESCRIPTION
**Pull request by SachinPNikam**
hello,
I udated the Blazor Example for EF core as well as mock .so we can use Database or mock based on startup code.
Thank  you

### CODE REVIEW

**Comment by SachinPNikam** [comment]
attched person table script, let me know if the helpful , Thanks 
[personTable.zip](https://github.com/MarimerLLC/csla/files/4138876/personTable.zip)

**Comment by rockfordlhotka** [comment]
Thank you for the PR, I'm on holiday this week and plan to review PRs sometime next week.

File: Samples/BlazorExample/BlazorExample/DataAccess/PersonDal.cs [hunk 1]
```diff
@@ -1,74 +1,74 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
+﻿//using System;
```
Comment (rockfordlhotka) [inline Samples/BlazorExample/BlazorExample/DataAccess/PersonDal.cs]: Shouldn't this file just be deleted/removed - no sense leaving it here commented out.

File: Samples/BlazorExample/BlazorExample/Server/Startup.cs [hunk 2]
```diff
@@ -23,7 +24,14 @@ public void ConfigureServices(IServiceCollection services)
                   new[] { "application/octet-stream" });
       });
 
-      services.AddTransient(typeof(DataAccess.IPersonDal), typeof(DataAccess.PersonDal));
+      //services.AddTransient(typeof(DataAccess.IPersonDal), typeof(DataAccess.PersonDal));
```
Comment (rockfordlhotka) [inline Samples/BlazorExample/BlazorExample/Server/Startup.cs]: Is there a generic `AddTransient`? Like `AddTransient<DataAccess.IPersonDal, DataAccess.Mock.PersonDal>()`?

**Comment by rockfordlhotka** [comment]
@SachinPNikam Nice changes, I like this a lot!

Just a couple comments - one about deleting a file instead of commenting out the contents, the other about `AddTransient` and whether we can make that code slightly nicer.

Can you respond to those? Thank you!

**Comment by rockfordlhotka** [comment]
Also, you should probably put that table script into the project folder and add it to the repo so it is with the code.

File: Samples/BlazorExample/BlazorExample/DataAccess/PersonDal.cs (hunk 1 above)
Comment (SachinPNikam) [reply Samples/BlazorExample/BlazorExample/DataAccess/PersonDal.cs]: Sure I will correct this .

File: Samples/BlazorExample/BlazorExample/Server/Startup.cs (hunk 2 above)
Comment (SachinPNikam) [reply Samples/BlazorExample/BlazorExample/Server/Startup.cs]: I will try

**Comment by SachinPNikam** [comment]
as per your suggestions I did  PR for  Person table scripts in sample project, deleted old PersonDal .also now blazor sample default conneting to the mock .only  generic AddTransient related remains .
Thank you !

**Pull request merged**